packages/grafana-prometheus/src/query_hints.ts

```diff
@@ -230,6 +210,125 @@ export function getInitHints(datasource: PrometheusDatasource): QueryHint[] {
   return hints;
 }
 
+export function getExpandRulesHints(query: string, mapping: RuleQueryMapping): QueryHint[] {
+  const hints: QueryHint[] = [];
+  const mappingForQuery = Object.keys(mapping).reduce((acc, ruleName) => {
+    if (query.search(ruleName) === -1) {
+      return acc;
+    }
+
+    if (mapping[ruleName].length > 1) {
+      const { idx, expandedQuery, identifier, identifierValue } = getRecordingRuleIdentifierIdx(
+        query,
+        ruleName,
+        mapping[ruleName]
+      );
+
+      // No identifier detected add warning
+      if (idx === -1) {
+        hints.push({
+          type: 'EXPAND_RULES_WARNING',
+          label: 'Query contains a recording rule. To be able to expand it please add an identifier label/value.',
+        });
```

Comment:
@catherineymgui @ellisda Please comment on this text.

Reply:
```suggestion
        hints.push({
          type: 'EXPAND_RULES_WARNING',
          label: 'Query contains recording rules and multiple matches found. To be able to expand it please add an identifier label/value.',
        });
```

The previous message is pretty terse, just "Query contains recording rules" so I'm not sure how much additional verbosity fits... :thinking: 

I think the key is to say that we matched or found "multiple recording rules"

I think your last sentence is good.

Reply:
I think it's good to make it as short as possible but to the point. How about this? 

> We found multiple recording rules in this query. To expand the recording rule, add an identifier label/value.

I removed the word please and made it active voice [as per these guidelines](https://docs.google.com/document/d/1Vx6EHVDTOpz_sUmM5Mtkx3XTMEj0bHen6mmStnp1IR4/edit)

Reply:
I think that works. Keeping with active voice, I think "We found multiple recording rules that match in this query" might be better. The "match" is the key here. But I like your current suggestion as well.

FWIW there is a difference between:
1. finding multiple recording rules in the query `metric1 - metric2` - where both match a recording rule
2. finding a single recording rule in the query `metric` that matches with multiple recording rule definitions that each record into a metric named `metric`